static/erdblick/features.js

```diff
@@ -6,23 +6,26 @@ import {uint8ArrayFromWasm, uint8ArrayToWasm} from "./wasm.js";
  * Bundle of a WASM TileFeatureLayer and a rendered representation
  * in the form of a Cesium PrimitiveCollection.
  *
- * The WASM TileFatureLayer object is stored as a blob when not needed,
+ * The WASM TileFeatureLayer object is stored as a blob when not needed,
  * to keep the memory usage within reasonable limits. To use the wrapped
  * WASM TileFeatureLayer, use the peek()-function.
  */
 export class FeatureTile
 {
 // public:
+    forceShow;
```

Comment:
What is it for?

Reply:
Also noticed it, removed it in performance improvements PR :)